courses/machine_learning/deepdive2/supplemental/solutions/autoencoder.ipynb

```diff
@@ -0,0 +1,687 @@
+{
```

Comment:
<ul><li>GAN -&gt; autoencoder</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/836/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>fixed here and throughout. Thanks!</p>